web_widget_one2many_tags/static/src/js/web_widget_one2many_tags.js

```diff
@@ -209,4 +210,4 @@ openerp.web_widget_one2many_tags = function(instance)
             return this._super(record, column);
         },
     });
-}
+});
```

Comment:
you need to return the classes this addons creates here in order to have then accessible for other addons